Documentation/compatibility/WCF-Improved-WCF-chain-trust-certificate-validation-for-Net.Tcp-certificate-authentication.md

```diff
@@ -0,0 +1,33 @@
+## Improved WCF chain trust certificate validation for Net.Tcp certificate authentication
+
+### Scope
+Minor
+
+### Version Introduced
+.NET 4.7.2 and serviced on all .NET 3.5+ released versions via windows security roll-up
+
+### Source Analyzer Status
+NotPlanned
+
+### Change Description
+Improved chain trust certificate validation when using certificate authentication with transport security with WCF. With this improvement client certificates that are used to authenticate to a server must be configured for Client Authentication.  Similarly server certificates that are for the authenticating a server must be configured for Server Authentication. With this change if the root certificate is disabled, the certificate chain validation will fail. 
+
+This change is on by default, and can be turned off by config setting.
+
+### Recommended Action
+  - Validate if your server and client certification has the required EKU, if not, update your certification
```

Comment:
EKU, if not --> EKU. If not, 
period after certification
should EKU be EKU OID attributes, or something similar?